troubadix/plugins/valid_oid.py

```diff
@@ -122,8 +146,10 @@ def check_content(
                     )
                 return
 
-            # Fixed OID-scheme for SUSE SLES OS OIDs
-            elif "1.3.6.1.4.1.25623.1.1.4." in oid:
+            # TBD: 3 is n/a in Confluence
+            # elif vendor_number == "3":
```

Comment:
Maybe already throw a warning, that an OID with a probably not available vendor has been found?

Reply:
Mhhh, indeed we could just drop that for now until it is clarified what vendor should be using this OID range.

```suggestion
```